### PR TITLE
Add natMap for Sentinel configuration

### PR DESCRIPTION
**Pull request by fgiannetti**
I have the same issue that was in cluster, but in sentinel configuration.
Hope it helps
Thanks!

### CODE REVIEW

**Comment by mrniko** [comment]
Thanks for contribution!

**Comment by fgiannetti** [comment]
Hey @mrniko!
This change was merged but is not included in the 3.11.0 version code. 
That was some problem with it?
Thanks!

**Comment by mrniko** [comment]
@fgiannetti 

Sorry, my mistake! Please apply changes against `master` branch and not 3.0.0

**Comment by fgiannetti** [comment]
@mrniko 
Thats ok! No problem ;)
I create a new PR against master as you said (https://github.com/redisson/redisson/pull/2137)
Again, thank you!

File: redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java [hunk 1]
```diff
@@ -438,7 +442,10 @@ public void operationComplete(Future<List<Map<String, String>>> future) throws E
 
     private String createAddress(String host, Object port) {
         if (host.contains(":") && !host.startsWith("[")) {
-            host = "[" + host + "]";
+        	String[] splittedHost= host.split(":");
+            host = "[" + applyNatMap(splittedHost[0]) + splittedHost[1] + "]";
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java]: IPv6 format should be taken in account

Comment (fgiannetti) [reply redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java]: fixed on https://github.com/redisson/redisson/pull/2137

**Pull request merged**